workspace/user.go

```diff
@@ -4,15 +4,22 @@ import (
 	"fmt"
 	"io"
 
+	"github.com/pkg/errors"
+
 	"github.com/astronomer/astro-cli/houston"
 	"github.com/astronomer/astro-cli/pkg/printutil"
+	"github.com/astronomer/astro-cli/user"
 )
 
 // Add a user to a workspace with specified role
-func Add(workspaceId, email, role string, client *houston.Client, out io.Writer) error {
+func Add(workspaceID string, email, role string, client *houston.Client, out io.Writer) error {
+	if !user.IsValidEmail(email) {
+		return errors.New(email + " is an invalid email address")
+	}
+
 	req := houston.Request{
 		Query:     houston.WorkspaceUserAddRequest,
-		Variables: map[string]interface{}{"workspaceId": workspaceId, "email": email, "role": role},
+		Variables: map[string]interface{}{"workspaceId": workspaceID, "email": email, "role": role},
```

Comment:
also this change breaks graphql query please revert it back:
https://github.com/astronomer/astro-cli/blob/99d2ef08ecc3c512129fc073d92ac233f69edc6d/houston/queries.go#L352